editmenu/CopyFunction.java

```diff
@@ -0,0 +1,19 @@
+package editmenu;
+
+import java.awt.event.ActionEvent;
+import java.awt.event.ActionListener;
+
+import javax.swing.JTextPane;
+
```

Comment:
A general comment explaining purpose and function of this class would be good, even if name gives a general idea